src/box/iproto_constants.h

```diff
@@ -49,255 +57,270 @@ enum {
 	 *      ...
 	 * ]
-	 */
-	IPROTO_METADATA = 0x32,
-	IPROTO_BIND_METADATA = 0x33,
-	IPROTO_BIND_COUNT = 0x34,
-	/** Position of last selected tuple in response. */
-	IPROTO_POSITION = 0x35,
-
-	/* Leave a gap between response keys and SQL keys. */
-	IPROTO_SQL_TEXT = 0x40,
-	IPROTO_SQL_BIND = 0x41,
+	 */					\
+	_(IPROTO_METADATA, 0x32)		\
+	_(IPROTO_BIND_METADATA, 0x33)		\
+	_(IPROTO_BIND_COUNT, 0x34)		\
+	/** Position of last selected tuple in response. */			\
+	_(IPROTO_POSITION, 0x35)		\
+						\
+	/* Leave a gap between response keys and SQL keys. */			\
+	_(IPROTO_SQL_TEXT, 0x40)		\
+	_(IPROTO_SQL_BIND, 0x41)		\
 	/**
 	 * IPROTO_SQL_INFO: {
-	 *     SQL_INFO_ROW_COUNT: number
+	 *      SQL_INFO_ROW_COUNT: number
 	 * }
-	 */
-	IPROTO_SQL_INFO = 0x42,
-	IPROTO_STMT_ID = 0x43,
-	/* Leave a gap between SQL keys and additional request keys */
-	IPROTO_REPLICA_ANON = 0x50,
-	IPROTO_ID_FILTER = 0x51,
-	IPROTO_ERROR = 0x52,
+	 */					\
+	_(IPROTO_SQL_INFO, 0x42)		\
+	_(IPROTO_STMT_ID, 0x43)			\
+	/* Leave a gap between SQL keys and additional request keys */		\
+	_(IPROTO_REPLICA_ANON, 0x50)		\
+	_(IPROTO_ID_FILTER, 0x51)		\
+	_(IPROTO_ERROR, 0x52)			\
 	/**
 	 * Term. Has the same meaning as IPROTO_RAFT_TERM, but is an iproto
 	 * key, rather than a raft key. Used for PROMOTE request, which needs
 	 * both iproto (e.g. REPLICA_ID) and raft (RAFT_TERM) keys.
-	 */
-	IPROTO_TERM = 0x53,
-	/** Protocol version. */
-	IPROTO_VERSION = 0x54,
-	/** Protocol features. */
-	IPROTO_FEATURES = 0x55,
-	/** Operation timeout. Specific to request type. */
-	IPROTO_TIMEOUT = 0x56,
-	/** Key name and data sent to a remote watcher. */
-	IPROTO_EVENT_KEY = 0x57,
-	IPROTO_EVENT_DATA = 0x58,
-	/** Isolation level, is used only by IPROTO_BEGIN request. */
-	IPROTO_TXN_ISOLATION = 0x59,
-	/** A vclock synchronisation request identifier. */
-	IPROTO_VCLOCK_SYNC = 0x5a,
+	 */					\
+	_(IPROTO_TERM, 0x53)			\
+	/** Protocol version. */		\
+	_(IPROTO_VERSION, 0x54)			\
+	/** Protocol features. */		\
+	_(IPROTO_FEATURES, 0x55)		\
+	/** Operation timeout. Specific to request type. */			\
+	_(IPROTO_TIMEOUT, 0x56)			\
+	/** Key name and data sent to a remote watcher. */			\
+	_(IPROTO_EVENT_KEY, 0x57)		\
+	_(IPROTO_EVENT_DATA, 0x58)		\
+	/** Isolation level, is used only by IPROTO_BEGIN request. */		\
+	_(IPROTO_TXN_ISOLATION, 0x59)		\
+	/** A vclock synchronisation request identifier. */			\
+	_(IPROTO_VCLOCK_SYNC, 0x5a)		\
 	/**
 	 * Name of the authentication method that is currently used on
 	 * the server (value of box.cfg.auth_type). It's sent in reply
 	 * to IPROTO_ID request. A client can use it as the default
 	 * authentication method.
-	 */
-	IPROTO_AUTH_TYPE = 0x5b,
-	/*
-	 * Be careful to not extend iproto_key values over 0x7f.
-	 * iproto_keys are encoded in msgpack as positive fixnum, which ends at
-	 * 0x7f, and we rely on this in some places by allocating a uint8_t to
-	 * hold a msgpack-encoded key value.
-	 */
-	IPROTO_KEY_MAX
-};
+	 */					\
```

Comment:
Why do you put a backslash here but don't put it on other lines?

Reply:
After preprocessing the whole comment will be reduced to one whitespace. That's why it's enough to put line continuation symbol only at the end of a multiline comment. But without it, the comment will become just an empty line without line continuation, and the preprocessor will stop parsing `define` there.

Reply:
Thanks, I didn't know about such nuances.